framework/graph/GraphBuilder.kt

```diff
@@ -212,16 +220,21 @@ class GraphBuilder(
     }
 
     fun explain(vertex: Vertex.Thing) {
-        NotificationService.launchCompletableFuture(Service.notification, LOGGER) {
-            val iterator = graph.reasoning.explanationIterators[vertex]
-                ?: runExplainQuery(vertex).also { graph.reasoning.explanationIterators[vertex] = it }
-            fetchNextExplanation(vertex, iterator)
-        }.exceptionally { e -> Service.notification.systemError(LOGGER, e, UNEXPECTED_ERROR) }
+        val explain = transactionSnapshot?.options()?.explain()?.get() ?: false
```

Comment:
Do you think it would improve consistency with the rest of Studio if we:

- rename `explain` to `canExplain`;
- split this method up into `tryExplain` and `explain`, where `tryExplain` does "if `canExplain` then `explain()` else `userWarning(EXPLAIN_NOT_ENABLED)`"?